src/domain/state.ts

```diff
@@ -941,33 +1053,13 @@ function getAppendixAgreements() {
       accepted: false,
     },
     {
-      name: APPENDIX_LARGE_SCALE_DATA_SHARING,
-      accepted: false,
-    },
-    {
-      name: APPENDIX_PREPUBLICATION_POLICY,
+      name: APPENDIX_DATA_ACCESS_POLICY,
       accepted: false,
     },
     {
-      name: APPENDIX_PUBLICATION_POLICY,
+      name: APPENDIX_IP_POLICY,
       accepted: false,
     },
-    {
-      name: APPENDIX_NIH_GENOMIC_INVENTIONS,
-      accepted: false,
-    },
-    {
-      name: APPENDIX_OECD_GENETIC_INVENTIONS,
-      accepted: false,
-    },
-    {
-      name: APPENDIX_CLOUD_SECURITY,
-      accepted: false,
-    },
-    {
-      name: APPENDIX_GA4GH_FRAMEWORK,
-      accepted: false,
-    }
```

Comment:
Found it! 🎉

Reply:
lol sorry forgot to point to the *actual* change with a comment! 🥚